crates/goose/src/agents/moim.rs

```diff
@@ -0,0 +1,64 @@
+use crate::agents::extension_manager::ExtensionManager;
+use crate::conversation::message::Message;
+use crate::conversation::Conversation;
+use rmcp::model::Role;
+
+/// The Minus One Info message provides ephemeral context that's included in LLM calls
+/// but never persisted to conversation history.
```

Comment:
that comment seems to be not related to the function, you know what I think fo taht